src/server.jsx

```diff
@@ -104,9 +45,16 @@ export default (locals) => {
             bundles.map((path) => <script key={path} src={path} />)
           )}
         </body>
-      </html>
-    </StaticRouter>
+      </StaticRouter>
+    </HelmetProvider>
   );
 
-  return '<!DOCTYPE html>' + renderedHtml;
+  const { helmet } = helmetContext;
+
+  const css = Object.keys(assets)
```

Comment:
We're still using assets data from server side rendering, which is a problem for offline as sw.js would cache assets from compilation of `webpack.prod.js`.